src/customnodesession.cpp

```diff
@@ -129,6 +128,7 @@ Status CustomNodeSession::createBlob(const struct CustomNodeTensor* tensor, Infe
         SPDLOG_LOGGER_ERROR(dag_executor_logger, "Node {}; session: {}; Unspecified output precision from custom node",
             this->getName(),
             this->getSessionKey());
+        this->releaseTensorResources(tensor, library);
```

Comment:
Looks to me that it woudl be better to have helper struct

TensorResourcesGuard {
void disarm() {
 disarmed = true;
}
~TensorResourcesGuard() {
if (!disarmed)
this->releaseRensorResources(tensor, library)
}
}

Reply:
Done